### PR TITLE
Added a step when you deploy AAD Connect to perform-G-Suite-migration.md

### PR DESCRIPTION
**Pull request by dariomws**
https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1525

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 1]
```diff
@@ -195,7 +191,9 @@ If your project doesn't already have all of the required APIs enabled, you must
...
+
+You can also deploy Azure Active Directory (Azure AD) Connect to provision your Mail Users, see [Deploy Microsoft 365 Directory Synchronization in Microsoft Azure](https://docs.microsoft.com/office365/enterprise/deploy-office-365-directory-synchronization-dirsync-in-microsoft-azure) for an overview, and [Set up directory synchronization for Microsoft 365](https://docs.microsoft.com/office365/enterprise/set-up-directory-synchronization) for set up instructions. Then, you need to deploy an Exchange server in your on-premises environment for user management, and mail-enabled your users using this server. For more information, see [How and when to decommission your on-premises Exchange servers in a hybrid deployment](https://docs.microsoft.com/exchange/decommission-on-premises-exchange) and [Manage mail users](/Exchange/ExchangeServer/recipients/mail-users.md).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:196]:
```suggestion
You can also deploy Azure Active Directory (Azure AD) Connect to provision your Mail Users, see [Deploy Microsoft 365 Directory Synchronization in Microsoft Azure](https://docs.microsoft.com/office365/enterprise/deploy-office-365-directory-synchronization-dirsync-in-microsoft-azure) for an overview, and [Set up directory synchronization for Microsoft 365](https://docs.microsoft.com/office365/enterprise/set-up-directory-synchronization) for setup instructions. Then, you need to deploy an Exchange server in your on-premises environment for user management, and mail-enable your users using this server. For more information, see [How and when to decommission your on-premises Exchange servers in a hybrid deployment](https://docs.microsoft.com/exchange/decommission-on-premises-exchange) and [Manage mail users](/Exchange/ExchangeServer/recipients/mail-users.md).
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:196]: Please change "set up instructions" to "setup instructions" (setup as an adjective one word as per Microsoft Style Guide), and I think "mail-enabled your users" should be "mail-enable your users".

Comment (dariomws) [reply Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:196]: Agree, it's mail enable in this case.

"mail-enable your users"

"mail-enabled user/s"

Comment (dariomws) [reply Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md]: Thanks for your help!

**Pull request merged**